api/agent/call.go

```diff
@@ -49,13 +49,23 @@ type Param struct {
 }
 type Params []Param
 
+const (
+	CloudEventHeader = "FN_CLOUD_EVENT"
+)
+
 func FromRequest(a Agent, app *models.App, path string, req *http.Request) CallOpt {
 	return func(c *call) error {
 		route, err := a.GetRoute(req.Context(), app.ID, path)
 		if err != nil {
 			return err
 		}
 
+		// Check whether this is a CloudEvent, if coming in via HTTP router (only way currently), then we'll look for a special header
+		if req.Header.Get(CloudEventHeader) != "" {
```

Comment:
the preferred method here https://github.com/cloudevents/spec/blob/master/http-transport-binding.md#22-data-attribute seems to be to use the `Content-Type` header `application/cloudevent`